torch/quantization/fx/quantization_patterns.py

```diff
@@ -400,14 +401,18 @@ def convert(self, quantizer, node, load_arg, debug=False):
 @register_quant_pattern(torch.nn.functional.instance_norm)
 @register_quant_pattern(torch.nn.functional.layer_norm)
 @register_quant_pattern(torch.nn.functional.leaky_relu)
+@register_quant_pattern(torch.sigmoid)
+@register_quant_pattern('sigmoid')
+@register_quant_pattern('sigmoid_')
 class DefaultNode(QuantizeHandler):
     ''' Common quantized op, first input and first output will be quantized
     '''
     def convert(self, quantizer, node, load_arg, debug=False):
         if not self.all_nodes:
             return NotImplemented
-        assert node.op in ['call_module', 'call_function'], 'Only call_module and ' + \
-            'call_function are handled in DefaultNode'
+        assert node.op in ['call_module', 'call_function', 'call_method'], 'Only call_module and ' + \
```

Comment:
Should the text of the assert change to reflect that call methods are supported?

Reply:
sure